client/components/lists/listBody.js

```diff
@@ -36,10 +36,43 @@ BlazeComponent.extendComponent({
     }
 
     if ($.trim(title)) {
+      // Parse for @user and #label mentions, stripping them from the title
+      // and applying the appropriate users and labels to the card instead.
+      const currentBoard = Boards.findOne(Session.get('currentBoard'));
+
+      // Find all @-mentioned usernames, collect a list of their IDs and strip
+      // their mention out of the title.
+      let foundUserIds = []; // eslint-disable-line prefer-const
+      currentBoard.members.forEach((member) => {
+        const username = Users.findOne(member.userId).username;
+        const nameNdx = title.indexOf(`@${username}`);
+        if(nameNdx !== -1) {
+          foundUserIds.push(member.userId);
+          title = title.substr(0, nameNdx)
```

Comment:
Maybe there could be a race condition here if the title is modified between the index capture and the substring (unlikely I admit). Any reason to not use

```js
title = title.replace(`@${username}`, '');
```

That seems more clear to me.